jenkins/internal-ci/generic/pr/cortx_deployment/rgw-integration.groovy

```diff
@@ -0,0 +1,261 @@
+
+                    sh encoding: 'utf-8', label: 'cortx-rgw-integration RPMS', returnStdout: true, script: """
+                        rm -rvf /root/build_rpms
+                        mkdir -p "/root/build_rpms"
+                        shopt -s extglob
+                        if ls ./dist/*.rpm; then
+                            cp ./dist/!(*.src.rpm|*.tar.gz) "/root/build_rpms"
+                        fi
+                        rpm -qi createrepo || yum install -y createrepo
+                        createrepo -v .
+                    """
+                }
+            }   
+        }
+
+        // Release cortx deployment stack
+        stage('Release') {
+            steps {
+				script { build_stage = env.STAGE_NAME }
+
+                dir('cortx-re') {
+                    checkout([$class: 'GitSCM', branches: [[name: '*/main']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'CloneOption', depth: 1, honorRefspec: true, noTags: true, reference: '', shallow: true], [$class: 'AuthorInChangelog']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', url: 'https://github.com/Seagate/cortx-re']]])
+                }
+
+                //Install tools required for release process
+                sh label: 'Installed Dependecies', script: '''
+                    yum install -y expect rpm-sign rng-tools python3-pip
+                    # ln -fs $(which python3.6) /usr/bin/python2 
+                    # systemctl start rngd
+                '''
+
+                // Integrate components rpms
+                sh label: 'Collect Release Artifacts', script: '''
+                    
+                    rm -rf "${DESTINATION_RELEASE_LOCATION}"
+                    mkdir -p "${DESTINATION_RELEASE_LOCATION}"
+                    if [[ ( ! -z `ls /root/build_rpms/*.rpm `)]]; then
+                        mkdir -p "${CORTX_ISO_LOCATION}"
+                        cp /root/build_rpms/*.rpm "${CORTX_ISO_LOCATION}"
+                    else
+                        echo "RPM not exists !!!"
+                        exit 1
+                    fi 
+                    pushd ${COMPONENTS_RPM}
+                        for component in `ls -1 | grep -v "rgw-integration"`
+                        do
+                            echo -e "Copying RPM's for $component"
+                            if ls $component/last_successful/*.rpm 1> /dev/null 2>&1; then
+                                cp $component/last_successful/*.rpm "${CORTX_ISO_LOCATION}"
+                            fi
+                        done
+                    popd
+                    # Symlink 3rdparty repo artifacts
+                    ln -s "${THIRD_PARTY_DEPS}" "${THIRD_PARTY_LOCATION}"
+                        
+                    # Symlink python dependencies
+                    ln -s "${PYTHON_DEPS}" "${PYTHON_LIB_LOCATION}"
+                '''
+
+                // sh label: 'RPM Signing', script: '''
```

Comment:
Remove `rpm signing` block. We do not need it for internal tests

Reply:
Done.